docs/pages/blog/docs-updates.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 'New documentation structure: better navigate MUI's products'
```

Comment:
What do you think about something less formal and more conversational? I'm trying to come up with a way to center the reader in the title.

```suggestion
title: 'Our docs just got a major upgrade—here's what that means for you'
```